.buildscript/configure-compose.gradle

```diff
@@ -19,7 +19,7 @@ android {
     compose true
   }
   composeOptions {
-    kotlinCompilerVersion "1.4.0-dev-withExperimentalGoogleExtensions-20200720"
```

Comment:
Like this change!

Reply:
Oops, according to Google devs this is still required, albeit with a different value. But this compiled, so not sure why.